### ISSUE TITLE
Missing Dutch accessibility translations

### ISSUE THREAD

**Issue by JorenVos** [Instabug/Instabug-React-Native#1240]
There are missing translations when using iOS VoiceOver/Android TalkBack and Instabug is set to Dutch (NL). 

- iOS: accessibility hint keys are being read out
- Android: English accessibility hints are being read out

See attached video.

https://github.com/Instabug/Instabug-React-Native/assets/10380088/54d081b9-d75e-4994-98f5-736269913b44

### Steps to Reproduce the Problem

Set language to Dutch. Activate VoiceOver/TalkBack.

### Expected Behavior

Dutch accessibility hints

### Actual Behavior

English accessibility hints (Android)/Keys (iOS)

### SDK Version

13.1.1

**Comment by JorenVos** [comment]
Bump.
